docs/advanced-topics/message-system/custom-messages.md

```diff
@@ -5,14 +5,14 @@ description: A brief explanation of Custom Messages use in Netcode for GameObjec
 ---
 import ImageSwitcher from '@site/src/ImageSwitcher.js';
 
-If you don't want to use the Netcode for GameObjects (Netcode) messaging system, you don't have to. You can use a thin layer called "Custom Messages" to implement your own messaging behavior and/or add custom targeting. They're unbound to any game object. Custom messages can be used in combination with [RPC messages](../messaging-system.md). 
+If you don't want to use the Netcode for GameObjects (Netcode) messaging system, you don't have to. You can use a thin layer called "Custom Messages" to implement your own messaging behavior or add custom targeting. They're unbound to any GameObject. You can use Custom messages with [RPC messages](../messaging-system.md). 
 
 There are two types of custom messages:
 - Unnamed
 - Named
 
 ## Unnamed Messages
-Unnamed messages can be thought of as a single sending channel. A message sent has one receive handler, this is useful for building your own custom messaging system. Netcode for GameObjects handles delivering and receiving custom unnamed messages, you can provide your own "custom message headers" to determine the type of unnamed message being sent or received.
+You can think about named messages as a single sending channel. A message sent has one receive handler, which is useful for building your own custom messaging system. Netcode for GameObjects handles delivering and receiving custom unnamed messages. You can create your own custom message headers to find the unnamed message types.
```

Comment:
> You can think about unnamed messages as if you are sending information over a single unique channel.  There is only one receiver handler per unnamed message, which can useful for building your own custom messaging system where you can define your own message headers. Netcode for GameObjects handles delivering and receiving custom unnamed messages, you determine what kind of information you want to transmit over the channel.

Maybe something like the above?

Reply:
Much better. I'll just make a few tweaks  🙂. 

> You can think about unnamed messages as if you are sending information over a single unique channel. There's only one receiver handler per unnamed message, which can help when building a custom messaging system where you can define your own message headers. Netcode for GameObjects handles delivering and receiving custom unnamed messages; you determine what kind of information you want to transmit over the channel.